lib/s3_helper.rb

```diff
@@ -23,6 +23,10 @@ def initialize(s3_bucket_name, tombstone_prefix = '', options = {})
       end
   end
 
+  def self.get_bucket_and_folder_path(s3_bucket_name)
+    return s3_bucket_name.downcase.split("/".freeze, 2)
```

Comment:
You don't need the `return` in Ruby when it's the last statement ;)

Reply:
Wow, I did that again :man_facepalming: 
Fixed, thanks!